kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -145,6 +153,7 @@
     public LinearLayout layout;
     public boolean isTakePhoto;
     public boolean isAttachment;
+    public boolean isImageVideo;
     public Integer currentConversationId;
```

Comment:
Method names start with is and get, if variable name is isImage than what we will give as method name, 
isIsImage will sound weird.

Reply:
i was following the convention that was earlier used in the code. Refer to line 154, 155 (isAttachment, isTakePhoto).
Will change it though.

Reply:
No need to copy mistakes

Reply:
done